test/helpers/cilium.go

```diff
@@ -309,6 +309,19 @@ func (s *SSHMeta) GetEndpointsIDMap() (map[string]string, error) {
 	return endpoints.KVOutput(), nil
 }
 
+// GetAllEndpointsIds returns a mapping of all Docker container name to to its
+// corresponding endpoint ID, and an error if the list of endpoints cannot be
+// retrieved via the Cilium CLI.
+func (s *SSHMeta) GetAllEndpointsIds() (map[string]string, error) {
```

Comment:
I'm not sure why you add this function instead of using `GetEndpointsIds` could you explain why?

Reply:
filter := `{range [?(@.status.labels.security-relevant[0]!="reserved:health")]}{@.status.external-identifiers.container-name}{"="}{@.id}{"\n"}{end}`

Since we do not have labels, this `status.labels.security-relevant[0]` does not work.

I will not be changing this as part of this PR.

Reply:
Make sense. Thanks